testsuite/features/step_definitions/cobbler_steps.rb

```diff
@@ -254,15 +254,37 @@
 end
 
 Then(/^the local logs for Cobbler should not contain errors$/) do
+  node = get_target('server')
+
+  # normal log file
+  cobbler_log_file = '/var/log/cobbler/cobbler.log'
+  local_file = '/tmp/cobbler.log'
+  # to avoid a race condition with "tar" as called by "mgrctl cp", we need to work on a copy:
```

Comment:
I would prefer we fix this in the command itself but good enough for now I guess.

Reply:
Cedric asked us to fix it first in the test suite if possible. Otherwise, he can take another look.